src/BlazorUI/Bit.BlazorUI/Styles/Fluent/bit.blazorui.fluent-dark.scss

```
@@ -9,3 +9,9 @@
 @import "shapes.fluent.scss";
 
 @import "typography.fluent.scss";
+
+@import "motion.fluent.scss";
```

Stylelint (17.11.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)